lib/node_modules/@stdlib/blas/base/srotg/docs/repl.txt

```diff
@@ -0,0 +1,26 @@
+
+{{alias}}( a, b )
+    Constructs a Givens rotation from two single-precision 
+    floating-point numbers.
+
+    Parameters
+    ----------
+    a: float
+        Rotational elimination parameter.
+    
+    b: float
+        Rotational elimination parameter.
+
+    Returns
+    -------
+    out: Float32Array
+        Computed values.
+
+    Examples
+    --------
+    // Standard usage:
+    > var out = {{alias}}( 0.0, 2.0 )
+    <Float32Array>[ 2.0, 1.0, 0.0, 1.0 ]
+
+    See Also
```

Comment:
We should also document the `assign` API, as it is part of the public interface.